source/sink-connector/fundamentals/error-handling-strategies.txt

```diff
@@ -88,19 +88,9 @@ option:
 .. warning:: Ordered Bulk Writes Can Result in Skipped Messages
 
    If you set your connector to tolerate errors and use ordered bulk writes, you
-   may lose data. When an ordered bulk write fails, the connector skips the
-   remaining writes in that batch and resumes processing the next batch.
-   
-   If you set your connector to tolerate errors and use unordered bulk writes,
-   you lose less data. Your connector attempts to perform every write in the
-   batch even if some writes fail. 
-   
-   When using unordered bulk writes, ensure you can perform the write operations
-   within each batch in any order.
-
-   To configure your connector to use unordered bulk writes, set the
-   ``bulk.write.ordered`` property to ``false``. To learn more about this
-   property, see the :ref:`<sink-configuration-message-processing>` page.
+   may lose data. If you set your connector to tolerate errors and use unordered bulk writes,
+   you lose less data. To learn more about bulk write operations, see
```

Comment:
Maybe something like this? Feel free to change the wording.

```suggestion
   the connector will perform all write operations that do not result in an error. To learn more about bulk write operations, see
```

Reply:
just changed the wording/structure altogether